argus/resources/windows/installCBinit.ps1

```diff
@@ -1,5 +1,6 @@
 param
 (
+    [string]$MsiWebLocation,

```

Comment:
I think we could add a default value for the `MsiWebLocation` to the previous value `http://www.cloudbase.it/downloads`